docs/topic/billing/reports.md

```diff
@@ -0,0 +1,82 @@
+# Billing reports
+
+## Billing reports in the web console
+
+Cloud providers provide a billing console accessible via the web, often
+with very helpful reports. While not automation-friendly (see next section
+for more information on automation), this is very helpful for doing ad-hoc
+reporting as well as trying to optimize cloud costs. For dedicated clusters,
+this may also be directly accessible to hub champions, allowing them to
+explore their own usage.
+
+### GCP
+
+You can get a list of all billing accounts you have access to via the [billing console](https://console.cloud.google.com/billing)
+(make sure you are logged in to the correct Google account). Alternatively,
+you can select 'Billing' from the left sidebar in the Google Cloud Console
+after selecting the correct project.
+
+Once you have selected a billing account, you can access a reporting
+interface by selecting 'Reports' on the left sidebar. This lets you group
+costs in different ways, and explore that over different time periods.
+Grouping and filtering by project and service (representing
+what kind of cloud product is costing us what) are the *most useful* aspects
+here. You can also download whatever you see as a CSV, although programmatic
+access to most of the reports here is unfortunately limited.
+
+### AWS
+
+For AWS, "Billing and Cost Management" is accessed from specific AWS
+accounts. If using consolidated billing, it must be accessed from the
+'management account' for that particular organization (see previous section
+for more information).
+
+You can access "Billing and Cost Management" by typing "Billing and Cost
+Management" into the top search bar once you're logged into the correct
+AWS account. AWS has a million ways for you to slice and dice these reports,
+but the most helpful place to start is the "Cost Explorer Saved Reports".
+This provides 'Monthly costs by linked account' and 'Monthly costs by
+service'. Once you open those reports, you can further filter and group
+as you wish. A CSV of reports can also be downloaded here.
```

Comment:
Missing link on `here`

Reply:
The `here` is a reference to the page the document is describing. I've clarified.